pcm_fitModelGroup.m

```diff
@@ -1,240 +1,240 @@
+%                   Commonly these are the scanning run #s for beta
+%                   regressors. If 1 vector is given, then partitions are 
+%                   assumed to be the same across subjects 
+%
+%   conditionVec: {#Subjects} Cell array with condition assignment vector
+%                   for each subject. Rows of conditionVec{subj} define
+%                   condition assignment of rows of Y{subj}. If 1 vector is given, 
+%                   then conditions are  assumed to be the same across subjects
+%                   If the (elements of) conditionVec are matrices, it is
+%                   assumed to be the design matrix Z, allowing the
+%                   specification individualized models. 
+%--------------------------------------------------------------------------
+% OPTION:
+%   'runEffect': How to deal with effects that may be specific to different
+%                imaging runs:
+%                  'random': Models variance of the run effect for each subject
+%                            as a seperate random effects parameter.
+%                  'fixed': Consider run effect a fixed effect, will be removed
+%                            implicitly using ReML.
+%   'fitScale':    Fit additional scaling parameter for each of the
+%                  subjects? Defaults to 1. This makes a lot of sense, if the scaling of the 
+%                  data is not of the same intensity across subjects. 
+%                  However, when you want to test strongly against a null model that
+%                  does not predict any difference between different conditions, 
+%                  then the scaling parameter makes the alternative model
+%                  more flexible  
+%   'isCheckDeriv: Check the derivative accuracy of theta params. Done using
+%                  'pcm_checkderiv'. This function compares input to finite
+%                  differences approximations. See function documentation.
+%
+%   'MaxIteration': Number of max minimization iterations. Default is 1000.
+%
+%   'verbose':      Optional flag to show display message in the command
+%                   line (e.g., elapsed time). Default is 1.
+% 
+%   'S',S         : Structure of the NxN noise covariance matrix -
+%                   otherwise independence is assumed
+% 
+%--------------------------------------------------------------------------
+% OUTPUT:
+%   T:      Structure with following subfields:
+%       SN:             Subject number
+%       likelihood:     Group-fit likelihood (with subj included)
+%       noise:          Noise Variance 
+%       scale:          Scale parameter for each subject (if fitScale set to 1)  
+%       run:            Run variance (if runEffect = 'random'); 
+%
+%    theta_hat:  Estimated parameters at the overall fitting (including
+% <<<<<<< HEAD
+%                noise and scale parameters). A mx1 cell array   
+% =======
+%                noise, scale and run parameters).  
+% >>>>>>> develop

```

Comment:
(not yours but...
shouldn't the diff marks be fixe, rather than commented out ?

Reply:
Actually I think @johanna-zumer had fixed that in her previous pull request because the current version of this file on github does not have anymore. I must have forgotten to update my version. Sorry about that!